.eslintrc.js

```diff
@@ -85,6 +85,8 @@ module.exports = {
     '@typescript-eslint/prefer-return-this-type': 'warn',
     '@typescript-eslint/require-await': 'warn',
     '@typescript-eslint/restrict-template-expressions': 'off',
+    // This is more performant; see https://v8.dev/blog/fast-async.
```

Comment:
please add this comment to the right side of the rule, so that keep-sorted is happy